### DISCUSSION TITLE
Integration with Authelia+LLdap

### DISCUSSION THREAD

**Discussion by Alumni1506** [9p4/jellyfin-plugin-sso#123] [Q&A] [answered]
I've setup a Authelia and LLdap to protect jellyfin and other services. I've had to remove authelia from jellyfin for 2 reasons.

1.  After authenticating with authelia, you still have to login again directly into jellyfin. So 2 login pages for the user.
2.  Due to having 2 login pages, authentication does not work with mobile or tv apps . I believe it's because there is no means to throw up a web browser for authelia to work?

What i'd to know is if i install jellyfin-plugin-sso, will the problem mentioned above go away?

**Comment by 9p4** [comment, marked as answer]
Kind of...

This SSO plugin still requires a web-browser for the authentication flow (but does get rid of the "log in twice" issue). However, because the plugin does not block all requests and instead implements a new endpoint for login, then you can use this plugin with other clients as long as they support [quick connect](https://jellyfin.org/docs/general/server/quick-connect/)

**Reply by Alumni1506** [reply]
Lastly, there's a requirement to develop a gui for the plugin. Is this still required if I am using authelia since it uses a webpage alread?

**Reply by 9p4** [reply]
there is no requirement to develop a gui to use the plugin
